transport.go

```diff
@@ -41,3 +46,22 @@ func (m *Manager) Register(s *grpc.Server) {
 func (m *Manager) Transport() raft.Transport {
 	return raftAPI{m}
 }
+
+func (m *Manager) Close() error {
+	m.connectionsMtx.Lock()
+	defer m.connectionsMtx.Unlock()
+
+	err := errCloseErr
+	for _, conn := range m.connections {
+		closeErr := conn.clientConn.Close()
+		if closeErr != nil {
+			errors.Wrap(err, closeErr.Error())
```

Comment:
This seems like a weird way to chain error messages. Let's use something like:

* https://pkg.go.dev/github.com/Jille/errchain#Append
* https://pkg.go.dev/github.com/hashicorp/go-multierror or
* https://pkg.go.dev/go.uber.org/multierr

Reply:
Fair enough. I changed over to hashicorp/go-multierror.

Are you comfortable with the defined base error, and returning nil if it hasn't changed? I figured it would be useful context to embed `error closing connections` at the error root, but I can take it out.

Reply:
Fine by me